content/docs/reference-react-dom.md

```diff
@@ -55,6 +55,21 @@ createPortal(child, container)
 ```
 
 Creates a portal. Portals provide a way to [render children into a DOM node that exists outside the hierarchy of the DOM component](/docs/portals.html).
+
+### `flushSync()` {#flushsync}
+
+```javascript
+flushSync(callback)
+```
+
+Force React to flush updates synchronously. This method is useful for being able to read the value of `state` immediately.
+
+> Note:
+> 
+> `flushSync` can have a significant impact on performance. Use sparingly.
+> 
+> `flushSync` may also run pending effects and synchronously apply any updates they contain before returning.
```

Comment:
It can also worse Suspense boundaries to show a "fallback" state when they otherwise wouldn't have.